index 7.js

```diff
@@ -0,0 +1,135 @@
+"use strict"
+
+const items = [
+    {
+        title: "Airpod",
+        price: 139.99,
+        quantity: 6,
+        isEnable: true,
+    },
+    {
+        title: "Iphone 14 pro max",
+        price: 1299,
+        quantity: 2,
+        isEnable: true,
+    },
+    {
+        title: "MSI Laptop",
+        price: 2599,
+        quantity: 4,
+        isEnable: true,
+    },
+    {
+        title: "Samsung Galaxy A73",
+        price: 348,
+        quantity: 12,
+        isEnable: true,
+    },
+    {
+        title: "Sony Playstation 5",
+        price: 499.99,
+        quantity: 1,
+        isEnable: true,
+    }
+];
+
+const soldItems = [];
+
+function quantityOfEachItem() {
+    items.forEach((item) => {
+        console.log(`${item.title} : ${item.quantity}`);
+    });
+}
+
+// quantityOfEachItem();
+
+function insertNewItem(title, price, quantity, isEnable) {
+    items.push({
+        title,
+        price,
+        quantity,
+        isEnable
+    });
+    console.log(`${title} has been successfully added`);
+}
+
+// insertNewItem("Gaming Mouse",200,5,false);
+
+function sell(itemTitle) {
+    for (const item of items) {
+        if (itemTitle.toLowerCase() === item.title.toLowerCase() && item.quantity > 0) {
```

Comment:
By using `id` there is no need to do this.

Reply:
Yeah that's right, I'll fix it.